Define some common default settings typically used on most
systems.

Index: etc/make.conf
===================================================================
--- etc.orig/make.conf
+++ etc/make.conf
@@ -295,7 +295,7 @@ PORT_LOGDIR="/var/log/ebuilds"
 # PORTAGE_NICENESS provides a default increment to emerge's niceness level.
 #     Note: This is an increment. Running emerge in a niced environment will
 #     reduce it further. Default is unset.
-#PORTAGE_NICENESS=3
+PORTAGE_NICENESS="10"
 #
 # AUTOCLEAN enables portage to automatically clean out older or overlapping
 #     packages from the system after every successful merge. This is the
@@ -378,7 +378,7 @@ FEATURES="
 # CCACHE_SIZE sets the space limitations for ccache. The default size is
 #     "2G", or 2 gigabytes.  Units are specified with 'G', 'M', or 'K'.
 #
-#CCACHE_SIZE="512M"
+CCACHE_SIZE="2G"
 #
 # CCACHE_DIR sets the ccache path.  If not specified, portage will default
 #     to "${PORTAGE_TMPDIR}/ccache".
@@ -471,12 +471,12 @@ PORTAGE_ELOG_SYSTEM="save"
 # Gentoo's config protection. Such files will never be
 # overwritten by an update without giving the admin
 # a chance to merge the old and new file contents.
-#CONFIG_PROTECT=""
+CONFIG_PROTECT="/usr/local/etc"
 
 # Where to install HTML documentation symlinks.
 # Those links will be free of a version number and therefore
 # stay the same for all package versions.
-#DOC_SYMLINKS_DIR=""
+DOC_SYMLINKS_DIR="/usr/share/doc/html"
 
 # User and e-mail for Gentoo's changelog authoring tool.
 # Only required if you are actually using that tool.
@@ -511,10 +511,10 @@ PORTAGE_ELOG_SYSTEM="save"
 #LINGUAS=""
 
 # The command used to compress documentation during the install phase.
-#PORTAGE_COMPRESS="bzip2"
+PORTAGE_COMPRESS="lzma"
 
 # Flags for the PORTAGE_COMPRESS command.
-#PORTAGE_COMPRESS_FLAGS="-9"
+PORTAGE_COMPRESS_FLAGS="-9"
 
 # If this is set, it should contain the path to a GnuPG public key ring.
 # The keys in that keyring will then be used for cryptographic
@@ -527,7 +527,7 @@ PORTAGE_ELOG_SYSTEM="save"
 # place-holder that will be substituted with an integer pid. For example, a
 # value of "ionice -c 3 -p \${PID}" will set idle io priority. For more
 # information about ionice, see ionice(1). This variable is unset by default.
-#PORTAGE_IONICE_COMMAND="ionice -c 3 -p \${PID}"
+PORTAGE_IONICE_COMMAND="ionice -c 3 -p \${PID}"
 
 # When a given file does not exist in DISTDIR, search for the file in this
 # list of directories. Search order is from left to right.
@@ -553,12 +553,17 @@ PORTAGE_ELOG_SYSTEM="save"
 # in the LAN; looking there first before downloading from the Internet.
 # This will typically be the /usr/portage/distfiles directory of some
 # other machine, exported as a network share.
-#PORTAGE_RO_DISTDIRS="/usr/local/portage/distfiles"
+PORTAGE_RO_DISTDIRS="
+ /usr/portage/local/distfiles/local
+ /usr/portage/local/distfiles
+ /usr/portage/local/distfiles/precious
+ /usr/portage/local/distfiles/mnt
+"
 
 # Override the working directory permissions when Portage
 # unpacks and compiles source files.
 # By default, no other users can look into the working directory.
-#PORTAGE_WORKDIR_MODE="0705"
+PORTAGE_WORKDIR_MODE="0705"
 
 # Several packages create digital X.509 PKI preset certificates when they
 # are installed.
@@ -566,15 +571,15 @@ PORTAGE_ELOG_SYSTEM="save"
 # You can specify default settings for such certificates here. Note that
 # in the absence of appropriate default settings here, cryptographically
 # weak algorithms or key lengths might be chosen by the installer.
-#SSL_BITS="2048"
+SSL_BITS="2048"
 #SSL_COMMONNAME="First Last"
 #SSL_COUNTRY="AT"
-#SSL_DAYS="730"
-#SSL_EMAIL="root@localhost"
+SSL_DAYS="730"
+SSL_EMAIL="root@localhost"
 #SSL_LOCALITY="City"
 #SSL_ORGANIZATION="My Company"
 #SSL_STATE="State"
-#SSL_UNIT="Preliminary usage"
+SSL_UNIT="Preliminary usage"
 
 # If you are using X11, setting this variable will restrict
 # the video adapter cards to be compiled in.
